wafer/pages/views.py

```diff
@@ -49,3 +52,10 @@ def slug(request, url):
         return EditPage.as_view()(request, pk=page.id)
 
     return ShowPage.as_view()(request, pk=page.id)
+
+
+class PageViewSet(viewsets.ModelViewSet):
+    """API endpoint for users."""
+    queryset = Page.objects.all()
+    serializer_class = PageSerializer
+    permission_classes = (DjangoModelPermissions, )
```

Comment:
Probably. Internal details like "exclude_from_static" and "show_in_menu" are currently only visible via the admin interface, which is why I didn't use that, but I don't see any significant implications to making that information available.